doc/user/content/get-started.md

```diff
@@ -5,216 +5,271 @@ menu: "main"
+     If you re-run the `SELECT` statement at different points in time, you can see the updated results based on the latest data.
 
-1. From within the CLI, create a source from the `wikirecent` file:
+1. To see the sequence of updates affecting the results over time, you can use `TAIL`:
 
     ```sql
-    CREATE SOURCE wikirecent
-    FROM FILE '[path to wikirecent]' WITH (tail = true)
-    FORMAT REGEX '^data: (?P<data>.*)';
+    COPY (TAIL avg_bid) TO stdout;
     ```
 
-    This source takes the lines from the stream, finds those that begins with `data:`, and then captures the rest of the line in a column called `data`
+    To cancel out of the stream, press **CTRL+C**.
 
-    You can see the columns that get generated for this source:
+## What's next?
 
-    ```sql
-    SHOW COLUMNS FROM wikirecent;
-    ```
+That's it! You just created your first materialized view on top of streaming data. We encourage you to continue exploring the PubNub source using the supported [SQL commands](/sql/). Here are a few examples of other interesting patterns to try:
 
-1. Because this stream comes in as JSON, we'll need to normalize the data to perform aggregations on it. Materialize offers the ability to do this easily using our built-in [`jsonb` functions](/sql/functions/#json).
+{{< expand Joins >}}
 
-    ```sql
-    CREATE MATERIALIZED VIEW recentchanges AS
-        SELECT
-            val->>'$schema' AS r_schema,
-            (val->'bot')::bool AS bot,
-            val->>'comment' AS comment,
-            (val->'id')::float::int AS id,
-            (val->'length'->'new')::float::int AS length_new,
-            (val->'length'->'old')::float::int AS length_old,
-            val->'meta'->>'uri' AS meta_uri,
-            val->'meta'->>'id' as meta_id,
-            (val->'minor')::bool AS minor,
-            (val->'namespace')::float AS namespace,
-            val->>'parsedcomment' AS parsedcomment,
-            (val->'revision'->'new')::float::int AS revision_new,
-            (val->'revision'->'old')::float::int AS revision_old,
-            val->>'server_name' AS server_name,
-            (val->'server_script_path')::text AS server_script_path,
-            val->>'server_url' AS server_url,
-            (val->'timestamp')::float AS r_ts,
-            val->>'title' AS title,
-            val->>'type' AS type,
-            val->>'user' AS user,
-            val->>'wiki' AS wiki
-        FROM (SELECT data::jsonb AS val FROM wikirecent);
-    ```
-
-1. From here we can start building our aggregations. The simplest place to start is simply counting the number of items we've seen:
+Joins are one of the most powerful features in SQL, but something quite hard to get right in the streaming world. Materialize efficiently supports [all types of SQL joins](/sql/join/#examples) under all the conditions you would expect from a traditional relational database.
+
+1. Create and populate a table with static, reference data:
```

Comment:
Do we need to create a table here? Can't we use the PubNub stream?

Reply:
There isn't really anything to join with there, unless you do something weird like a self-join.

Reply:
That os the main thing with showing a join, you need some sort of reference data. We could potentially define a set as a sub-query, but this could be confusing.